CodeStarterCamp_Week1/main.swift

```diff
@@ -13,73 +13,53 @@ func drawPeperoBody(body: String, topping: String) {
     print(topping + body + topping)
 }
 
-func drawPeperoBar(lenght: Int, topping: String, body: String) {
+func drawPeperoBodyAndLength(length: Int, topping: String, body: String) {
     if body == "***" {
-        for _ in 1...lenght {
+        for _ in 1...length {
             drawPeperoBody(body: body, topping: topping)
         }
     }
     else {
```

Comment:
요기도 else 가 밑에 내려가 있군요!
```suggestion
    } else {
```